File: hw/ramos/u1/HW11ChickenCoops/Python/ChickenCoop.py
class ChickenCoop:
    def __init__(self, id):
        self.id = id
        self.chickens = []

    def add_chicken(self, chicken):
        self.chickens.append(chicken)

    def __str__(self):
        chickens_info = "\n  ".join(str(chicken) for chicken in self.chickens)
        return f"ChickenCoop(id={self.id})\n  {chickens_info}"
    

File: hw/ramos/u1/HW11ChickenCoops/JavaScript/ChickenCoop.js
import Chicken  from "./Chicken.js";

export default class ChickenCoop {
  constructor(id) {
    this.id = id;
    this.chickens = [];
  }

  addChicken(chicken) {
    this.chickens.push(chicken);
  }

  toString() {
    const chickensInfo = this.chickens.map(chicken => "  " + chicken.toString()).join("\n");
    return `ChickenCoop(id=${this.id})\n${chickensInfo}`;
  }
}
